### ISSUE TITLE
How to get user profile(name, profile picture)

### ISSUE THREAD

**Issue by rjravijoshi111** [hungdev/react-native-instagram-login#64]
I m using the latest version but I m getting a success response  with this data

{"access_token":"IGQVJXVE15WEpxY2ZAlSkldfsfcxqV3M3NWVhVmZAMb2NrT29WcWZAIU3FWWjdoUUNaU0tRTjsdasdhYZATVHVmxQOWlBMnVJejRkbGss9iZAmxDTHN3aUp4SEdOTmZAYa0pBT3RTb0ZAPenpOcGo2asdUXVxdDczasdaQUxlUTQ3QkhFZA1ktTkNFN2RfNWJv","user_id":178xxxxxxxxxx} 

 but I need the user's name and profile picture, so could you please help me how to get this information

**Comment by slauzinho** [comment]
After you get the token you just need to use Instagram Graphapi https://developers.facebook.com/docs/instagram-api/

**Comment by rjravijoshi111** [comment]
@slauzinho I have use Instagram graph api but in there I have getting only username.

But I need users full name, profile image.

**Comment by Dhanraj-bidchat** [comment]
**Can I make this API call with this node module:**
 GET \
 "https://graph.facebook.com/v3.2/17841405309211844?fields=business_discovery.username(bluebottle){followers_count,media_count,media{comments_count,like_count}}&access_token={access-token}"

**Comment by SrinivasanKannan18** [comment]
> After you get the token you just need to use Instagram Graphapi https://developers.facebook.com/docs/instagram-api/

Yeah I have used that API but it shown 'Invalid OAuth access token.' error and error code:190

**Comment by AdeelMughal** [comment]
> @slauzinho I have to use Instagram graph API but in there I have got the only username. I m using this API [https://graph.instagram.com/${data.user_id}?fields=${'id,username'}&access_token=${data.access_token}](https://graph.instagram.com/$%7Bdata.user_id%7D?fields=$%7B'id,username'%7D&access_token=$%7Bdata.access_token%7D)
> 
> But I need the user's full name, profile image.

Any update on this ? I am still looking for solution. I am unable to get user's profile picture, email and full name.
